dev-docs/bidders/appnexus.md

```diff
@@ -43,4 +55,33 @@ var adUnits = [{
 }]
 {% endhighlight %}
 
-(Sizes set in `adUnit` object will also apply to the AppNexus bid requests.)
+{: .alert.alert-info :}
+Sizes set in the `adUnit` object will also apply to the AppNexus bid requests.
```

Comment:
Why do we need to break out these separate sections?  For each media type below, it seems like we're just redirecting back to the standard documentation.

Reply:
> Why do we need to break out these separate sections?

It's intended as an enhancement.  It's meant to provide another way for users to discover the features supported by the `appnexus` adapter beyond the section [Bidders with video and native demand](http://prebid.org/dev-docs/bidders.html#bidders-with-video-and-native-demand).  For example, you can scan the mini-TOC and see that we support native ads.

> For each media type below, it seems like we're just redirecting back to the standard documentation.

Yes.  The idea is that once you see in the mini-TOC that we support native, from there, you can jump to the native ads documentation, knowing that the AN adapter supports the features described there (rather than us having a miniature "native" section of content here with info that is redundant with 'Show Native Ads').

Once the multiformat feature/docs are out, I would like to add another little section to this mini-TOC that points the user to the multi-format docs.